src/librustc/session/config.rs

```diff
@@ -807,6 +809,7 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
                       `everybody_loops` (all function bodies replaced with `loop {}`).",
                      "TYPE"),
         opt::opt_u("", "show-span", "Show spans for compiler debugging", "expr|pat|ty"),
+        opt::opt("", "complete-at", "Give completions at the given position", "POS"),
```

Comment:
It is a codemap byte offset now, which is completely wrong. I plan to change it to filename:bytepos soon.